### ISSUE TITLE
[BUG] Nuclear Winter: Semi-reproducible crash on destroying clutter

### ISSUE THREAD

**Issue by EdHerdman** [ZDoom/Raze#864]
### Raze version

raze-x64-1.7pre-522-gacfca3fbd

### Which game are you running with Raze?

Duke Nukem 3D

### What Operating System are you using?

Windows 11

### Please describe your specific OS version

Windows 11 Pro Version 22H2 OS Build 22621.1105

### Relevant hardware info

AMD Ryzen 7 3700X, NVIDIA RTX 3060

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

I have experienced random-seeming crashes throughout Duke: Nuclear Winter, but I found one reproducible crash. In E2L6, Christmas Village, look inside the windows near the map start and note the clutter: White bags of mail. Proceed to break all of them and it is very likely Raze will crash.

I have tested this by kicking them one by one, by shotgun, by chaingun, and once by launching rockets from outside. The more bags broken, the more likely the bug will trigger. The crash doesn't necessarily trigger immediately on breaking the last bag, but can trigger after a couple seconds while the spawned letters fall. The game crashed a second or two after taking these screenshots, when I had only broken the bags seen here.
![Duke_0003](https://user-images.githubusercontent.com/96923667/213609547-9da6bc6e-adc2-49cf-baf9-9045fb852fb6.png)
![Duke_0004](https://user-images.githubusercontent.com/96923667/213609556-0d04258d-114f-422d-a6fe-b3dad7bb7177.png)
[CrashReport4.zip](https://github.com/ZDoom/Raze/files/10462904/CrashReport4.zip)

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Enter map E2L6
2. Look in windows of nearby office
3. Shoot mail bags, receive crash report

**Comment by coelckers** [comment]
I cannot reproduce this.

**Comment by EdHerdman** [comment]
In that case here's the crash report.
[CrashReport4.zip](https://github.com/ZDoom/Raze/files/10474093/CrashReport4.zip)

**Comment by EdHerdman** [comment]
Tried this again with the current version, [raze-x64-1.7pre-535-g13c9e9916](https://devbuilds.drdteam.org/raze/raze-x64-1.7pre-535-g13c9e9916.7z) and I still get a crash here. Used a pipe bomb to trigger it this time.
[CrashReport5.zip](https://github.com/ZDoom/Raze/files/10475930/CrashReport5.zip)

**Comment by FitzRoyX** [comment]
I happen to have a save near here and also couldn't get it to happen. Tried about 15 times. What is the hash of your grps?

**Comment by EdHerdman** [comment]
It's from Steam's Megaton Edition. The GRP hash:

Name: nwinter.grp
Size: 16169365 bytes (15 MiB)
SHA1: a6728f621f121f9db02ee67c39efdbb5eea95711

I have it in a "nwinter" directory with 12 files in total: E2L1.map to E2L8.map, nwactor.con, nwdes.con, nwinter.con, and nwinter.grp.

**Comment by EdHerdman** [comment]
Current findings: I can't trigger this in any CI build, but I get it reliably in builds from the DRD Team Development Builds page.
raze-x64-1.7pre-564-g17392fe40 from Feb 06 still crashes on me.
I tried Visual Studio 2022 builds going back three weeks (3c4ab95, 6c77d1c, 13c9e99, 1938efa, 8052bc7, 17392fe, ede52f2) - none crash. I tried Visual Studio 2019 Release 3c4ab95, and that also doesn't crash.

I'll try the next DRD Team dev build and see if it's gone there. I'll note that with the CI builds I'm copying over the missing files (the sound library DLLs, and the soundfonts folder) from the latest dev release which includes them, but maybe these files have something to do with it.

**Comment by EdHerdman** [comment]
I've tested raze-x64-1.7pre-578-g6729c342a and the bug no longer triggers. I tested the new build with the shotgun, chaingun, pipe bombs, and rockets from inside the room and without. Whatever it was, I think it's fixed now.

I believe I can rule out some kind of user configuration error as I'm running the same configuration on each portable version (extract files, copy in the game directories, and run) - and the bug still triggers in old builds.
